azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -691,6 +702,10 @@ func expandAzureRmKubernetesClusterAgentProfiles(d *schema.ResourceData) []conta
 		OsType:         containerservice.OSType(osType),
 	}
 
+	if maxPods := int32(config["max_pods"].(int)); maxPods > 0 {
+		profile.MaxPods = utils.Int32(maxPods)
+	}
```

Comment:
could we update the acceptance tests (for the internal network probably makes the most sense, rather than a new test?) to set this value?

Reply:
Done :)